.idea/runConfigurations/behave.xml

```diff
@@ -0,0 +1,16 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
ugh,let's not check in your personal .idea/blah files

Reply:
xml, puke

Reply:
I know, I know.  I did this as it actually has a use and there was a `.gitignore` run to explicitly ignore files in this directory which required me to `git add -f .idea/runConfigurations/behave.xml`.  I did this as the value is that you can run these feature files under behave after [attaching to the process]()https://www.jetbrains.com/help/pycharm/attaching-to-local-process.html) being ran from the command, `doit local`.  This allows for easier debugging than good ol' `gdb python subhub/app.py` in the case of a failing `behave` test.  This being said is an interesting talking point as to how we want to bias developers towards tooling.  I happily remove it but just wanted to articulate my rationale as to why.

Reply:
ok let's discuss next week

Reply:
Cool!